<launch>

  <arg name="namespace" default="/" />
  <arg name="respawn" default="False" />
  <arg name="use_proxy" default="False" />

  <node
    name="pot_states_osi_pi_publisher_node"
    pkg="smelter_osi_pi"
    type="slagsquare_pot_states_osi_pi_publisher_node.py"
    ns="$(arg namespace)"
    output="screen"
    clear_params="true"
    respawn="$(arg respawn)"
    respawn_delay="10">

    <rosparam
      file="$(find smelter_osi_pi)/config/umicore_osi_pi_config_params.yaml"
      command="load" />

    <rosparam
      file="$(find smelter_osi_pi)/config/umicore_pot_states_osi_pi_params.yaml"
      command="load" />

    <param name="~use_proxy" value="$(arg use_proxy)" />

  </node>

</launch>
